tests/endpoints/test_schemas.py

```diff
@@ -163,37 +156,12 @@ class SchemaExample(schemas.Model):
 
     assert m.import_data(
         {"around": {"origin": "40.730610,-73.935242", "offset": "1km", "scale": "2km", "decay": "0.1"}}
-    ).to_primitive() == {"around": {"origin": u"40.730610,-73.935242", "decay": 0.1, "scale": u"2km", "offset": u"1km"}}
+    ).to_primitive() == {"around": {"origin": "40.730610,-73.935242", "decay": 0.1, "scale": "2km", "offset": "1km"}}
 
     with pytest.raises(schemas.SchematicsDataError):
         m.import_data({"around": "40.730610,-73.935242"}, validate=True)
 
 
-def test_area_model():
```

Comment:
Is this still relevant to remove that test?

Reply:
Updated and brought back the test since we are using Area class now.